### ISSUE TITLE
GetMedia request results in JSON unmarshal error

### ISSUE THREAD

**Issue by grokify** [grokify/go-voicebase-v3#1]
When attempting to unmarshal response from `/v3/media` to [`VbMediaQueryResponse`](https://github.com/grokify/go-voicebase-v3/blob/master/vb_media_query_response.go), the following unmarshal error is encountered:

`panic: json: cannot unmarshal string into Go struct field VbMediaSummary.status of type voicebase.VbStatusEnum`

`VbMediaSummary.status` definition:

https://github.com/grokify/go-voicebase-v3/blob/master/vb_media_summary.go#L26

```go
Status VbStatusEnum `json:"status,omitempty"`
```

`VbStatusEnum` definition:

https://github.com/grokify/go-voicebase-v3/blob/6a2b2e67696b33cdacc9c4cbf39a4f93339f2d28/vb_status_enum.go

```go
type VbStatusEnum struct {
}
```

Swagger spec shows this to be a string:

https://github.com/grokify/go-voicebase-v3/blob/master/swagger_voicebase_v3.yaml#L68

```yaml
  VbStatusEnum:
    type: string
    enum:
    - accepted
    - failed
    - scheduled
    - running
    - finished
```

Example `VbMediaSummary.status` API response value is also a string: `"finished"`. This can be verified by running `examples/get_media_simple/get_media.go`

Looks like an error in Swagger Codegen.

**Comment by grokify** [comment]
This Swagger is built using the following Swagger Spec:

https://github.com/grokify/go-voicebase-v3/blob/8e410c1c07aa89d4fbe0e4f2cb486f83f706f3ca/swagger_voicebase_v3.yaml

**Comment by grokify** [comment]
Fixed in 504f493449ede61af17ef8aca971e2ddde0ba3da

**Comment by grokify** [comment]
This was an error using Swagger Codegen `2.2.3`. It should not occur in `2.3.0`
